deploy/infra/groups/data.bicep

```diff
@@ -46,12 +46,36 @@ param synapseMIStorageAccountRoles array = [
 ]
 param synapseMIPrincipalId string = ''
 
+// Name parameters for Postgres
+param serverName string = ''
+param administratorLogin string = ''
+param postgresAdminLoginPass string = '' 
+
+// Parameters with default values for Postgres
+param skuCapacity int = 2
+param skuName string = 'GP_Gen5_2'
+param skuSizeMB int = 51200
```

Comment:
again, change to string type, according to https://docs.microsoft.com/en-us/azure/templates/microsoft.dbforpostgresql/2017-12-01/servers?tabs=bicep. size type is string in the document.

Reply:
Changed to string.